docs/DOCKER.md

```diff
@@ -35,39 +35,61 @@ Instructions
 Otherwise, to build an existing environment with the most updated DB, remove DB volume by running `docker volume rm anyway_db_data`.
 Note - this will delete all of your local DB data!
 
-**5.** Start the container, go to the **anyway** directory and run:
+**5.** Anyway images stored on github package, to be able to pull the images you need to login to github using docker login.
+if you already logged in with docker to github source jump to the next step.
+
+#### **docker login**
+First, to login to github go to [this link](https://github.com/settings/tokens) of your github account and generate token with `read:packages` permission.
+
+Second, copy the token that you just generate to file and save the file, now run docker login command like this when `USERNAME` is your github username and `TOKEN.txt` is the file the stor the token.
+```bash
+$ cat TOKEN.txt | docker login docker.pkg.github.com -u USERNAME --password-stdin
```

Comment:
I think you can simplify here, just execute like this and it will prompt for the password, no need to save it in a txt file first:

```bash
docker login docker.pkg.github.com -u USERNAME
```